assets/scss/03-organisms/_emergency-alerts.scss

```
@@ -71,31 +105,14 @@ be included at the top of the page */
     @media ($bp-small-min) {
       display: none;
     }
-  }
-}
-
-//theme
-.ma__emergency-alerts {
-  position: relative;
-
-  &__header {
-    background-color: $c-warning;
-  }
-
-  &__content {
-    background-color: lighten($c-warning, 34%);
-  }
-
-  &__interface {
-    
     @media ($bp-small-max) {
       background-color: $c-warning;
       padding: 20px;
       padding-top: 0;
       transition: background-color .5s;
       
       .is-open & {
-        background-color: lighten($c-warning, 34%);
+        background-color: tint($c-warning, 34%);

```

I think this should be
```suggestion
        background-color: tint($c-warning, 90%);
```